File: 251010/특정 수 맞히기/catching-specific-number.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a = 0;

    while(a != 25) {
        cin >> a;

        if(a < 25) {
            cout << "Higher" << "\n";
        } else if(a > 25) {
            cout << "Lower" << "\n";
        }
    }

    cout << "Good";
    return 0;
}

File: 251104/특정 수 맞히기/catching-specific-number.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int a;
        while(true) {
            a = int.Parse(Console.ReadLine());
            if(a < 25) {
                Console.WriteLine("Higher");
            } else if(a > 25) {
                Console.WriteLine("Lower");
            } else if(a == 25) {
                Console.WriteLine("Good");
                break;
            }
        }
    }
}
